docs/instruction/03-runtime.md

```diff
@@ -1,215 +1,270 @@
+| eventmesh-connector-plugin | EventMesh Connector plugin module    |
+| eventmesh-runtime          | EventMesh Runtime runtime module      |
+| eventmesh-sdk-java         | EventMesh Java client SDK             |
+| eventmesh-starter          | EventMesh local startup project entry point |
+| eventmesh-spi              | EventMesh SPI loading module          |
 
-```console
-vim conf/eventmesh.properties
-```
+> Plugin modules follow the SPI specification defined by EventMesh, and custom SPI interfaces need to be annotated with `@EventMeshSPI`.
+>
+> Plugin instances need to be configured in the corresponding module under the `/main/resources/META-INF/eventmesh` directory with a mapping file for interface and implementation classes. The file name is the fully qualified class name of the SPI interface.
+>
+> The content of the file is the mapping from the plugin instance name to the plugin instance. For details, refer to the `eventmesh-connector-rocketmq` plugin module.
 
-Execute the `start.sh` script to start the EventMesh Runtime server.
+### 3.4 Plugin Explanation
 
-```console
-bash bin/start.sh
+#### 3.4.1 Install Plugins
+
+EventMesh has an SPI mechanism that allows EventMesh to discover and load plugins. There are two ways to install plugins:
+
+- Classpath loading: During local development, you can add dependencies in the `build.gradle` of the `eventmesh-starter` module. For example, to add the Kafka Connector plugin:
+
+```gradle
+dependencies {
+   implementation project(":eventmesh-runtime")
+    // Example: Add the Kafka Connector plugin
+   implementation project(":eventmesh-connectors:eventmesh-connector-kafka")
+}
 ```
 
-If you see "EventMeshTCPServer[port=10000] started...." , then the setup was successful.
+- File loading: By installing the plugin to the plugin directory, EventMesh will automatically load the plugins in the plugin directory based on certain conditions during runtime. Please refer to [2.3 Build](#23-build) and [2.4 Package Plugins](#24-package-plugins).
+
+>When you make changes to the source code, it is recommended to add the `build` task to the command provided in [2.3 Build](#23-build) to recompile and run unit tests. For example:
+>
+>```shell
+>./gradlew clean build dist -x spotlessJava -x generateGrammarSource --parallel --daemon
+>```
 
-![runtime_6](/images/install/runtime_6.png)
+#### 3.4.2 Use Plugins
 
-View the output log:
+EventMesh will load the plugins by default from the `dist/plugin` directory. You can change the plugin directory using `-DeventMeshPluginDir=your_plugin_directory`. The plugin instances needed at runtime can be configured in the `confPath` directory in the `eventmesh.properties` file. For example, by setting the following, you declare the use of the RocketMQ Connector plugin:
 
-```console
-cd /root/apache-eventmesh-1.9.0/logs
-tail -f eventmesh.out
+```properties
+# connector plugin
+eventMesh.connector.plugin.type=rocketmq
```

Comment:
eventmesh-connector-rocketmq does not run in the plugin way described above.

Reply:
Fixed in https://github.com/apache/eventmesh-site/pull/170/commits/dad1f5807f4146a7b06f8eb45e8cb350f5e7f6ae.